resources/lang/id/forum.php

```diff
@@ -0,0 +1,268 @@
+    'title' => 'osu!community',
+
+    'covers' => [
+        'create' => [
+            '_' => 'Set cover image',
+            'button' => 'Upload image',
+            'info' => 'Cover size should be at :dimensions. You can also drop your image here to upload.',
+        ],
+
+        'destroy' => [
+            '_' => 'Remove cover image',
+            'confirm' => 'Are you sure you want to remove the cover image?',
+        ],
+    ],
+
+    'email' => [
+        'new_reply' => '[osu!] New reply for topic ":title"',
+    ],
+
+    'forums' => [
+        'topics' => [
+            'empty' => 'No topics!',
+        ],
+    ],
+
+    'post' => [
+        'confirm_destroy' => 'Really delete post?',
+        'confirm_restore' => 'Really restore post?',
+        'edited' => 'Last edited by :user :when, edited :count times in total.',
+        'posted_at' => 'posted :when',
+
+        'actions' => [
+            'destroy' => 'Delete post',
+            'restore' => 'Restore post',
+            'edit' => 'Edit post',
+        ],
+    ],
+
+    'search' => [
+        'go_to_post' => 'Go to post',
+        'post_number_input' => 'enter post number',
+        'total_posts' => ':posts_count posts total',
+    ],
+
+    'topic' => [
+        'deleted' => 'deleted topic',
+        'go_to_latest' => 'view latest post',
+        'latest_post' => ':when by :user',
+        'latest_reply_by' => 'latest reply by :user',
+        'new_topic' => 'Post new topic',
+        'post_reply' => 'Post',
+        'reply_box_placeholder' => 'Type here to reply',
+        'started_by' => 'by :user',
+
+        'create' => [
+            'preview' => 'Preview',
+            // TL note: this is used in the topic reply preview, when
+            // the user goes back from previewing to editing the reply
+            'preview_hide' => 'Write',
+            'submit' => 'Post',
+
+            'placeholder' => [
+                'body' => 'Type post content here',
+                'title' => 'Click here to set title',
+            ],
+        ],
+
+        'jump' => [
+            'enter' => 'click to enter specific post number',
+            'first' => 'go to first post',
+            'last' => 'go to last post',
+            'next' => 'skip next 10 posts',
+            'previous' => 'go back 10 posts',
+        ],
+
+        'post_edit' => [
+            'cancel' => 'Cancel',
+            'post' => 'Save',
+        ],
+    ],
+
+    'topic_watches' => [
+        'index' => [
+            'title' => 'Forum Subscriptions',
+            'title_compact' => 'forum subscriptions',
+            'title_main' => 'Forum <strong>Subscriptions</strong>',
+
+            'box' => [
+                'total' => 'Topics subscribed',
+                'unread' => 'Topics with new replies',
+            ],
+
+            'info' => [
+                'total' => 'You subscribed to :total topics.',
+                'unread' => 'You have :unread unread replies to subscribed topics.',
```

Comment:
'unread' => 'Anda mempunyai :unread balasan yang belum dibalas di topik yang Anda langgan."
mungkin kata langgan terlihat sedikit janggal, ditunggu reply dari yang lain.

Reply:
Will change it to
>Anda mempunyai :unread balasan yang belum dibalas di topik langganan anda.